packages/instui-config/codemod-configs/v8/propNames.config.json

```diff
@@ -1,68 +1,44 @@
 {
   "Alert": {
-    "8.0.0": [
-      { "old": "closeButtonLabel", "new": "renderCloseButtonLabel" }
-    ]
+    "8.0.0": [{ "old": "closeButtonLabel", "new": "renderCloseButtonLabel" }]
   },
   "Avatar": {
     "8.0.0": [
-      { "old": "inline", "new": "display", "values": [
-        { "old": true, "new": "inline-block" },
-        { "old": false, "new": "block" }
-      ]},
+      {
+        "old": "inline",
+        "new": "display",
+        "values": [
+          { "old": true, "new": "inline-block" },
+          { "old": false, "new": "block" }
+        ]
+      },
       { "old": "variant", "new": "shape" }
     ]
   },
   "Breadcrumb.Link": {
-    "8.0.0": [
-      { "old": "icon", "new": "renderIcon" }
-    ]
-  },
-  "Button": {
-    "8.0.0": [
-      { "old": "buttonRef", "new": "elementRef" },
-      { "old": "fluidWidth", "new": "display", "values": [
-        { "old": true, "new": "block" },
-        { "old": false }
-      ]},
-      { "old": "icon", "new": "renderIcon" },
-      { "old": "variant", "new": "color", "values": [
-        { "old": "default" },
-        { "old": "light", "new": "primary-inverse" },
-        { "old": "ghost", "new": "primary" },
-        { "old": "ghost-inverse", "new": "primary-inverse" },
-        { "old": "icon" },
-        { "old": "icon-inverse", "new": "primary-inverse" },
-        { "old": "circle-default" },
-        { "old": "circle-primary", "new": "primary" },
-        { "old": "circle-danger", "new": "danger" }
-      ]}
-    ]
-  },
-  "CloseButton": {
```

Comment:
I've removed the Button and CloseButton codemods from here, people should use the new `updateV7Props` (as is in the button upgrade guide)